.env.defaults

```diff
@@ -5,3 +5,8 @@ JWTAUDIENCE=api.dev
 AWS_ACCESS_KEY_ID=XXXXXXXXXXXXXXXXXXXX
 AWS_SECRET_ACCESS_KEY=xxxxxxxxxxxxxxxxxxxx
 AWS_BUCKET=aws-bucket
+STORAGE_AMAZON_BUCKET=bucket-name
+STORAGE_AMAZON_REGION=bucket-region
+# HARBOR_NGINX_ENDPOINT=https://external.fqdn
```

Comment:
can you add a comment when exactly this env variable should be set? as it is by default commented it is not really clear for what it is used for.

Reply:
Now that we are automatically setting that `HARBOR_NGINX_ENDPOINT`, we can just remove it here, as well. :-)